deploy/storage/base_bigtable_info.yaml

```diff
@@ -1,13 +1,13 @@
 project: datcom-store
 instance: prophet-cache
 tables:
-  - auto2w_2023_06_13_14_16_23
-  - auto1w_2023_06_13_14_15_34
-  - auto1d_2023_06_13_14_15_34
-  - frequent_2023_06_04_09_17_02
+  - auto2w_2023_06_14_13_43_12
+  - auto1d_2023_06_19_21_00_50
+  - auto1w_2023_06_18_20_55_47
+  - frequent_2023_06_19_20_25_47
```

Comment:
Can we try this frequent_2023_06_15_20_08_36?

Reply:
Because cl/541059647 dropped the new temperature projectiuons from frequent but it didn't get built into ipcc.